### ISSUE TITLE
Further develop this package ?

### ISSUE THREAD

**Issue by behrica** [picasa/agridatasets#1]
Are you interested in further develop this package ?

I am interested in making the access to some JRC agri4cast datasets easier, namely:
 - the 25km grid itself
- JRC climate data (agro meteo)
- JRC crop modelling
- JRC pest host data (forest atlas)

and maybe add some reusable code on top of these:

- degree day calculations

**Comment by picasa** [comment]
Yes, some additions you suggested were on the TODO list : adding the grid and climate data.
I am unsure about to best way to do this, surely using some API will be better than directly including datasets (especially for climate data).

Maybe focusing on crop model inputs (soil, climate) would make more sense than a general "crop model" package (i.e. including outputs or convenience function for thermal time computation and so). Anyway, I did not thought this too much, just started a small package for my lab.
